helm_deploy/create-and-vary-a-licence-api/values.yaml

```diff
@@ -19,7 +19,7 @@ generic-service:
 
   # Environment variables to load into the deployment
   env:
-    JAVA_OPTS: "-Xmx512m"
+    JAVA_OPTS: "-Xmx768m"
```

Comment:
we tend to see OOM issues occasionally due to pulling back large caselists, we've now doubled the namespace limit to 2048 in CP - so have some more breathing room to increase this.